tests/test_ops/test_masked_conv2d.py

```diff
@@ -1,15 +1,55 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+import pytest
 import torch
 
+from mmcv.utils import IS_CUDA_AVAILABLE, IS_MLU_AVAILABLE
+
 
 class TestMaskedConv2d:
 
-    def test_masked_conv2d(self):
-        if not torch.cuda.is_available():
-            return
+    def _test_masked_conv2d(self, device):
         from mmcv.ops import MaskedConv2d
-        input = torch.randn(1, 3, 16, 16, requires_grad=True, device='cuda')
-        mask = torch.randn(1, 16, 16, requires_grad=True, device='cuda')
-        conv = MaskedConv2d(3, 3, 3).cuda()
+        input = torch.randn(1, 3, 16, 16, requires_grad=True, device=device)
+        mask = torch.randn(1, 16, 16, requires_grad=True, device=device)
+        if device == 'cuda':
```

Comment:
Could we use `conv = MaskedConv2d(3, 3, 3).to(device)`?

Reply:
Yep, your suggestion is very good, I have revised it.